cpp/foxglove-websocket/include/foxglove/websocket/websocket_server.hpp

```diff
@@ -149,6 +149,8 @@ class Server final : public ServerInterface<ConnHandle> {
                            const std::string& message);
   void broadcastTime(uint64_t timestamp) override;
   void sendServiceResponse(ConnHandle clientHandle, const ServiceResponse& response) override;
+  void sendServiceFailure(ConnHandle clientHandle, ServiceId serviceId, uint32_t callId,
```

Comment:
nit: it is weird that the method name doesn't match the server op, e.g. "sendServiceCallFailure"

Reply:
Agreed, but then I would also have to rename `sendServiceResponse` accordingly.